install.sh

```diff
@@ -73,11 +107,12 @@ main () {
     URL="https://github.com/opengrep/opengrep/releases/download/${VERSION}/${DIST}"
     echo
     echo "*** Installing Opengrep ${VERSION} for ${OS} (${ARCH}) ***"
-    echo
+    echo "${URL}"}
```

Comment:
I would leave as it was (no URL) as it's unlikely to be very useful.
Also: there's a stray `}`.

Reply:
Are you sure? Don't think so. Checked out the branch and got linter error.